src/main/java/User.java

```diff
@@ -8,19 +8,21 @@ public class User {
     private String password;
     private int id;
     private String email;
-    private Review reviews;
+    private List<Review> reviews;
     private List<Listing> listings;
     private Cart cart;
+    private int rating;
```

Comment:
does User need both reviews and a rating? it seems like it might eliminate confusion if we just calculate rating from the reviews. otherwise, what if someone passes an incorrect rating?